### DISCUSSION TITLE
.

### DISCUSSION THREAD

**Discussion by SbtrktCtrl** [apache/airflow#33183] [Q&A]

**Comment by potiuk** [comment]
> Dag deployed -> Dag-name-ID1
> Dag re-deployed -> Dag-name-ID2

What do you mean by that ? Can you elaborate and explain what you mean ? It's entirely unclear what you have done there.

**Reply by SbtrktCtrl** [reply]
.

**Reply by potiuk** [reply]
Can you post screenshots? I am not sure if that's the case but that is likely expected. When the DAG disappears, it is considers to be missing and we should not show the history of it. But you can open an issue (with scresnshots showoing the inconsistent behaviour - apparently the dag runs you see in browse are inconsistent)
